api/node/index.ts

```diff
@@ -305,7 +308,7 @@ export function loadFile(path: string) : Object {
         let errors = diagnostics.filter((d) => d.level == napi.DiagnosticLevel.Error);
 
         if (errors.length > 0) {
-            throw new CompileError("Could not compile " + path, errors);
+            throw new CompileError("Could not compile " + absoluteFilePath, errors);
```

Comment:
Given that making the path absolute is intended to be a workaround, I think here (and in the other place) it would be better to stick to using `path`.

Reply:
done